docs/integrations/sources/snowflake.md

```diff
@@ -23,73 +23,119 @@ The Snowflake source does not alter the schema present in your warehouse. Depend
 ### Requirements
 
 1. You'll need the following information to configure the Snowflake source:
-2. **Host**
+2. **Host (account name)**
 3. **Role**
 4. **Warehouse**
 5. **Database**
 6. **Schema**
 7. **Username**
-8. **Password**
-9. **JDBC URL Params** (Optional)
-10. Create a dedicated read-only Airbyte user and role with access to all schemas needed for replication.
-
+8. **Private key**
+9. **Password** 
+10. Create a dedicated read-only Airbyte user and role with access to all schemas needed for replication and add key-pair authentication 
```

Comment:
Could we add more information as to how to do it? With my limited Snowflake knowledge, I don't think I would know where to look for in order to do that

Reply:
I provided a detailed guide under this block ### Setup guide where i explained in details every step accordingly to snowflake doc with addition of details